kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -33,8 +33,20 @@ module.controller('grImageMetadataCtrl', [
 
     let ctrl = this;
 
+    ctrl.fieldAliases = window._clientConfig.fieldAliases;
     ctrl.showUsageRights = false;
     ctrl.usageRights = imageService(ctrl.image).usageRights;
+    ctrl.additionalMetadata = Object.fromEntries(
+      Object.entries(ctrl.image.data.aliases)
+        .map(([key, val]) => {
+          let match = ctrl.fieldAliases.find(_ => _.alias === key);
+          if (match) {
+            return [match.label, val];
+          } else {
+            return undefined;
```

Comment:
Would it be better to fall back to `[key, val]` here? we're not expecting it to happen, but it would IMHO be better to show an alias instead of a label than drop the field completely.

Reply:
Definitely makes sense to fallback to alias, thanks!